Code/Mantid/docs/source/algorithms/ConvertToReflectometryQ-v1.rst

```diff
@@ -121,6 +129,58 @@ Output:
     Qx Qz
     Ki Kf
     Pz_i + Pz_f Pz_i - Pz_f
+    
+Patch Plot with Dumped Vertexes
+###############################
+
+ConvertToReflectometryQ has the functionality to produce a table of vertexes before they are
+fitted to a normalised polygon. The plotting of these vertexes results in a patch plot that can be
+achieved by running the algorithm below.
+
+Usage
+-----
+**Example - Patch Plot using the Dumped vertexes from ConvertToReflectometryQ**
+
+.. code-block:: python
+
+    import numpy as np
```

Comment:
We want the full script. Include the part that does the full reduction (the script that I attached to your issue) as well as the bit that outputs the patch plot below.